pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/kms_vmware_overall/guide.en-gb.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Activate KMS with Hosted Private Cloud VMware on OVHcloud' 
+excerpt: 'Find out how to activate the OVHcloud KMS service to effectively secure your sensitive Hosted Private Cloud VMware on OVHcloud data' 
+updated: 2023-06-25
+---
+
+> [!warning]
+> 
+> OVHcloud KMS will soon be available in beta. This guide may be incomplete, and will be updated during the beta period. Feel free to send us your feedback on the dedicated Discord channel: <https://discord.gg/ovhcloud>.
+
+## Objective
+
+**This comprehensive guide explains how to configure and optimize the use of an OVHcloud Key Management Service (KMS) within your VMware on OVHcloud Hosted Private Cloud infrastructure.**
+
+## Requirements
+
+- An [OVHcloud customer account](/pages/account_and_service_management/account_information/ovhcloud-account-creation).
+- Have signed up to [VMware on OVHcloud](https://www.ovhcloud.com/fr/enterprise/products/hosted-private-cloud/){.external} solution. 
+- Have access to your Dedicated Cloud (VMware on OVHcloud) vSphere management interface.
+- Have sufficient IAM rights with your OVHcloud customer account and Hosted Private Cloud VMware on OVHcloud resources.
```

Comment:
Can you maybe details the step and IAM policies to be defined to give the proper right to a service account to use the KMS from VMWare ? Here or in the vmware_kms_vsphere_configuration?

Reply:
Absolutely @puKMS, do you have any knowledge if we have possible internal documentation relating the use of KMS with IAM and vSphere (if yes can you send it to me in pm ?).
Thanks :)